spelling.dic

```diff
@@ -1,6 +1,7 @@
 appsettings
 args
 async
+asynch
```

Comment:
Is this change necessary.

Reply:
It's not in this PR but it is in a class I was looking at. I generally add words when I find them, to avoiding having warnings in the editor in VS.